io/plugins/eu.esdihumboldt.hale.io.json.ui/src/eu/esdihumboldt/hale/io/json/ui/TypeSelectionPage.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2012 Data Harmonisation Panel
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
```

Comment:
I have replaced it by 

--------------
/*
 * Copyright (c) 2021 wetransform GmbH
 * 
 * All rights reserved. This program and the accompanying materials are made
 * available under the terms of the GNU Lesser General Public License as
 * published by the Free Software Foundation, either version 3 of the License,
 * or (at your option) any later version.
 * 
 * You should have received a copy of the GNU Lesser General Public License
 * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
 * 
 * Contributors:
 *     wetransform GmbH <http://www.wetransform.to>
 */
 ------------------
 but the same I have used seems to be used 1591 times in the project and the above one just 570 times. 
 Shouldn't use just one header everywhere?

Reply:
Well the year when the code was created is no constant and mentioning Fraunhofer and the data harmonisation panel as contributors is not correct. If you create a new class it will automatically add the header as configured in the current template.
The license is usually the same, though there are exceptions for some extensions that have a different license.